README.md

```diff
@@ -1,19 +1,233 @@
-The goal is to create a financial institution ie BANK and the project aims to impliment the following functions 
- How much money can the bank's cutomer take as a loan based on the customer's salary and income.The currency that the custmoer can get from the bank and the yearly interest.
-
-The following APIS are to be implimented
-
-1) GET: 
-This APLI is going to be used to get information from the bank's database , relating to information about the amounts that a customer can get as a loan and other necessary information.
-
-2) DELETE:
-This API is going to be implimented to delete information from the bank's record.
- Information like records on customers that have requested that their infor,ation should be removed. 
-
-
-3) POST:
-The API is expected to be implimented on saving the customer's information like name,adress,earnings into the bank's database. It is also gooing to be implimented to saveinformation related to loans that have been granted to the customer.
-
-
-4) PUT:
-This API is expected to be used on updating record that are saved by the bank 
+Contents
+Goal	2
+Database:	2
+Database Structure:	2
+Application Use:	3
+Log in Admin Account	3
+/Admin/admin login	3
+Admin registration	4
+/admin/create admin	4
+Users	5
+GET /customers	5
+Log in Customer account	7
+/customer/customer Login	7
+Customer registration:	8
+/customer/customer registration	8
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+Goal
+The goal of this project is to create the backend setup of a financial institution (Bank) application.
+
+Database:
+Database Structure:
+The project is going to be developing its database using PostgreSQL and the following schema is to be implemented:
+
+Customer:
+This table stores information about the bank�s customer. Contains essential information about the customer used to link the customer to their bank accounts and other services like loans. 
+Account:
+Used to store customer�s account details. Show the date the account was created or the account was delete and relevant information linking to the customer like customer id
+
+
+
+
+Transactions:

```

Comment:
It's table for one customer or for storage here information about all customers?